src/aml/tests/seabios_dsdt.rs

```diff
@@ -0,0 +1,307 @@
+    0x53, 0x08, 0x5f, 0x48, 0x49, 0x44, 0x0c, 0x41, 0xd0, 0x0c, 0x0f, 0x08, 0x5f, 0x55, 0x49, 0x44,
+    0x0a, 0x04, 0x08, 0x5f, 0x50, 0x52, 0x53, 0x11, 0x0e, 0x0a, 0x0b, 0x89, 0x06, 0x00, 0x09, 0x01,
+    0x09, 0x00, 0x00, 0x00, 0x79, 0x00, 0x14, 0x09, 0x5f, 0x53, 0x54, 0x41, 0x00, 0xa4, 0x0a, 0x0b,
+    0x14, 0x06, 0x5f, 0x44, 0x49, 0x53, 0x00, 0x14, 0x0b, 0x5f, 0x43, 0x52, 0x53, 0x00, 0xa4, 0x5f,
+    0x50, 0x52, 0x53, 0x14, 0x06, 0x5f, 0x53, 0x52, 0x53, 0x01, 0x10, 0x47, 0x0e, 0x5f, 0x53, 0x42,
+    0x5f, 0x14, 0x35, 0x43, 0x50, 0x4d, 0x41, 0x01, 0x70, 0x83, 0x88, 0x43, 0x50, 0x4f, 0x4e, 0x68,
+    0x00, 0x60, 0x70, 0x11, 0x0b, 0x0a, 0x08, 0x00, 0x08, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x61,
+    0x70, 0x68, 0x88, 0x61, 0x0a, 0x02, 0x00, 0x70, 0x68, 0x88, 0x61, 0x0a, 0x03, 0x00, 0x70, 0x60,
+    0x88, 0x61, 0x0a, 0x04, 0x00, 0xa4, 0x61, 0x14, 0x1a, 0x43, 0x50, 0x53, 0x54, 0x01, 0x70, 0x83,
+    0x88, 0x43, 0x50, 0x4f, 0x4e, 0x68, 0x00, 0x60, 0xa0, 0x05, 0x60, 0xa4, 0x0a, 0x0f, 0xa1, 0x03,
+    0xa4, 0x00, 0x14, 0x0a, 0x43, 0x50, 0x45, 0x4a, 0x02, 0x5b, 0x22, 0x0a, 0xc8, 0x5b, 0x80, 0x50,
+    0x52, 0x53, 0x54, 0x01, 0x0b, 0x00, 0xaf, 0x0a, 0x20, 0x5b, 0x81, 0x0c, 0x50, 0x52, 0x53, 0x54,
+    0x01, 0x50, 0x52, 0x53, 0x5f, 0x40, 0x10, 0x14, 0x4a, 0x06, 0x50, 0x52, 0x53, 0x43, 0x00, 0x70,
+    0x50, 0x52, 0x53, 0x5f, 0x65, 0x70, 0x00, 0x62, 0x70, 0x00, 0x60, 0xa2, 0x46, 0x05, 0x95, 0x60,
+    0x87, 0x43, 0x50, 0x4f, 0x4e, 0x70, 0x83, 0x88, 0x43, 0x50, 0x4f, 0x4e, 0x60, 0x00, 0x61, 0xa0,
+    0x0a, 0x7b, 0x60, 0x0a, 0x07, 0x00, 0x7a, 0x62, 0x01, 0x62, 0xa1, 0x0c, 0x70, 0x83, 0x88, 0x65,
+    0x7a, 0x60, 0x0a, 0x03, 0x00, 0x00, 0x62, 0x70, 0x7b, 0x62, 0x01, 0x00, 0x63, 0xa0, 0x22, 0x92,
+    0x93, 0x61, 0x63, 0x70, 0x63, 0x88, 0x43, 0x50, 0x4f, 0x4e, 0x60, 0x00, 0xa0, 0x0a, 0x93, 0x63,
+    0x01, 0x4e, 0x54, 0x46, 0x59, 0x60, 0x01, 0xa1, 0x08, 0x4e, 0x54, 0x46, 0x59, 0x60, 0x0a, 0x03,
+    0x75, 0x60, 0x10, 0x4e, 0x09, 0x5f, 0x47, 0x50, 0x45, 0x08, 0x5f, 0x48, 0x49, 0x44, 0x0d, 0x41,
+    0x43, 0x50, 0x49, 0x30, 0x30, 0x30, 0x36, 0x00, 0x14, 0x06, 0x5f, 0x4c, 0x30, 0x30, 0x00, 0x14,
+    0x15, 0x5f, 0x45, 0x30, 0x31, 0x00, 0x5c, 0x2f, 0x03, 0x5f, 0x53, 0x42, 0x5f, 0x50, 0x43, 0x49,
+    0x30, 0x50, 0x43, 0x4e, 0x46, 0x14, 0x10, 0x5f, 0x45, 0x30, 0x32, 0x00, 0x5c, 0x2e, 0x5f, 0x53,
+    0x42, 0x5f, 0x50, 0x52, 0x53, 0x43, 0x14, 0x06, 0x5f, 0x4c, 0x30, 0x33, 0x00, 0x14, 0x06, 0x5f,
+    0x4c, 0x30, 0x34, 0x00, 0x14, 0x06, 0x5f, 0x4c, 0x30, 0x35, 0x00, 0x14, 0x06, 0x5f, 0x4c, 0x30,
+    0x36, 0x00, 0x14, 0x06, 0x5f, 0x4c, 0x30, 0x37, 0x00, 0x14, 0x06, 0x5f, 0x4c, 0x30, 0x38, 0x00,
+    0x14, 0x06, 0x5f, 0x4c, 0x30, 0x39, 0x00, 0x14, 0x06, 0x5f, 0x4c, 0x30, 0x41, 0x00, 0x14, 0x06,
+    0x5f, 0x4c, 0x30, 0x42, 0x00, 0x14, 0x06, 0x5f, 0x4c, 0x30, 0x43, 0x00, 0x14, 0x06, 0x5f, 0x4c,
+    0x30, 0x44, 0x00, 0x14, 0x06, 0x5f, 0x4c, 0x30, 0x45, 0x00, 0x14, 0x06, 0x5f, 0x4c, 0x30, 0x46,
+    0x00,
```

Comment:
I think we could do this with an `include_bytes` macro probably.

Reply:
Fair enough. I don't like storing as a binary because it would need a hex editor to view the raw stream then, but I've put the actual data into a separate file as hex